internal/guest/runtime/hcsv2/uvm.go

```diff
@@ -292,6 +294,66 @@ func setupSandboxHugePageMountsPath(id string) error {
 	return storage.MountRShared(mountPath)
 }
 
+func getUserNameForID(spec *specs.Spec, uid uint32) (string, error) {
```

Comment:
we already have code that does this under `/internal/guest/runtime/hcsv2/spec.go`

Reply:
Removed duplicate